rasa/nlu/extractors/_entity_synonyms.py

```diff
@@ -0,0 +1,120 @@
+import os
```

Comment:
can we please add the comment on top here which we have for the other migrated components?